### DISCUSSION TITLE
Rename `Dropbox style` to `4 indent style`

### DISCUSSION THREAD

**Discussion by JavierSegoviaCordoba** [facebookincubator/ktfmt#172] [Ideas]
I think 4 indent style exists before `Dropbox` exists, so it is strange this naming because I would expect a specific configuration used by Dropbox company, but the fact is 4 indent style is the Kotlin convention. Another thing to change this name could be 'why anyone should know what is the style used by Dropbox?'

If someone installs the plugin or add `spotless`, should read the Readme to know what is Dropbox style, meanwhile it is just a "4 indent flag", so without reading the readme anyone could know what is using.

I can understand it could be named `Dropbox style` if it could have very specific configurations that are only used in Dropbox company.

**Comment by cortinico** [comment]
> I can understand it could be named `Dropbox style` if it could have very specific configurations that are only used in Dropbox company.

I'm also curious in knowing about this. I was not aware of the 4 spaces style being called the "Dropbox style"

**Comment by JavierSegoviaCordoba** [comment]
After talking with @cgrushko, dropbox-style is unmaintained and the idea is moving to Google Style (4/4 indents) and renaming it to something like kotlinlang-style.

IMO if it will match the official Kotlin conventions but with the strict/determinist formatting of ktfmt, it should be the default, and 2/2 should get a specific name. Because ktfmt hasn't reached 1.0, may be a good time to do it, if everyone agrees.

**Comment by djpeach** [comment]
For anyone wondering, this has been implemented. While dropbox style does still exist, there is a [list of styles](https://github.com/diffplug/spotless/blob/78a6dbc0b09a30270827b1dc94740c2124299fbd/plugin-gradle/src/main/java/com/diffplug/gradle/spotless/KotlinExtension.java#L151) to choose from now

**Comment by hick209** [comment]
I made a proposal for ktfmt 1.0.0 that is related to this discussion

See the proposal here https://github.com/facebook/ktfmt/discussions/451#discussioncomment-9305741

**Comment by hick209** [comment]
Closing this discussion as we're going to delete the dropbox style in favor of `kotlinlang` style
